fury/actor.py

```diff
@@ -123,8 +123,8 @@ def slicer(
         as slices. The key method of this object is ``display_extent`` where
         one can input grid coordinates and display the slice in space (or grid)
         coordinates as calculated by the affine parameter.
-
     """
+
```

Comment:
Why this change everywhere?  we prefer the opposite. (recommended by pycodestyle).  We are open to change that but you need to explain why. 

Thanks!